<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="1" hostname="DESKTOP-DO9NL9S" failures="1" tests="2" name="Default test(failed)" time="0.066" errors="0" timestamp="2020-06-02T20:34:38 IST">
  <testcase classname="test.TestNGFailedRetryDemo" name="test2" time="0.01"/>
  <testcase classname="test.TestNGFailedRetryDemo" name="test3" time="0.005">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at test.TestNGFailedRetryDemo.test3(TestNGFailedRetryDemo.java:25)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1510)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test3 -->
  <testcase classname="test.TestNGFailedRetryDemo" name="test1" time="0.0">
    <ignored/>
  </testcase> <!-- test1 -->
</testsuite> <!-- Default test(failed) -->
